src/Connect.svelte

```diff
@@ -34,18 +34,17 @@
   $: walletConnectState = config.walletConnect.state;
 </script>
 
-<button
-  on:click|stopPropagation={onConnect}
-  {style}
-  class="connect {className}"
+<Button
+  on:click={onConnect}
+  variant={buttonVariant}
   disabled={connecting}
   data-waiting={connecting || null}>
```

Comment:
Data properties won't make it through as-is

Reply:
Good catch! 4ebbf94d3205f573f786863e9a10c682c38a04d0